test/transaction/output.js

```diff
@@ -47,8 +56,30 @@ describe('Output', function() {
     newOutput.setScript(Script().add(0).toBuffer());
     newOutput.inspect().should.equal('<Output (0 sats) <Script: OP_0>>');
   });
-  
+
   it('has a inspect property', function() {
     output.inspect().should.equal('<Output (0 sats) <Script: >>');
   });
+
+  var output2 = new Output({
+    satoshis: 1100000000,
+    script: new Script('OP_2 21 0x038282263212c609d9ea2a6e3e172de238d8c39' +
+      'cabd5ac1ca10646e23fd5f51508 21 0x038282263212c609d9ea2a6e3e172de23' +
+      '8d8c39cabd5ac1ca10646e23fd5f51508 OP_2 OP_CHECKMULTISIG OP_EQUAL')
+  });
+  it('toBufferWriter', function() {
+    output2.toBufferWriter().toBuffer().toString('hex')
+      .should.equal('00ab904100000000485215038282263212c609d9ea2a6e3e172de2' +
+        '38d8c39cabd5ac1ca10646e23fd5f5150815038282263212c609d9ea2a6e3e172d' +
+        'e238d8c39cabd5ac1ca10646e23fd5f5150852ae87');
+  });
+  it('to/from JSON', function() {
```

Comment:
Please don't name tests plainly with the name of the method you're testing, so that the tests become more semantic. I'd suggest this test for example to be called `to/from JSON roundtrip works correctly`, with is what the test is actually looking for.